### PR TITLE
improve throttle

### PR DESCRIPTION
**Pull request by anasmatic**
it is much smoother now, hopefully will remove the throttle bumping effect the grader talked about in submit 1 result video.
the throttle diagram in submit one was like this : 
throttle_1 0_0 0_0 5

now it is smoother with little overshooting when we hit the max speed, but brakes on traffic lights is firm without jerking .
throttle_0 2_0 0005_0 05

### CODE REVIEW

**Comment by clockworkbits** [comment]
I tested it in both highway and test lot scenarios. Highway is fine, although I haven't waited till the end. Test lot is not - the car fails to stop at the last waypoint and in result drives in a loop.

**Comment by agotterba** [comment]
I ran it to the end of the simulator track, and it did stop successfully.  I was trying to test traffic light performance, but saw that there was too much lag, and detections were iffy with the parking lot classifier.  Once I switched to the simulator classifier, it stops at the stop-lines correctly, though the lag is even worse.

**Comment by anasmatic** [comment]
after using lowpass filter , throttle less smother but is ok , 
it can stop at the end waypoints , and traffic-lights with no issues 
throttle_1 0_0 0_0 5_f100

**Review by agotterba — APPROVED** [review]
I simulated it on the parking lot track, and see the car stop at the end of the waypoints now (and that it didn't stop when using the previous commit)

**Comment by anasmatic** [comment]
thank you for the approval : 

1 - if `is_dbw_enabled = False` 
then jerk at start will be around **500±** , 
Throttle graph shows that we start with highest value .
init_true_isdbw

2 - if `is_dbw_enabled = True` and wait for one minute before switch manual,
 then jerk at start will be around **200±** , 
Throttle graph shows that we start with gradual throttle to reach highest value ,
the more we wait the less time it takes to reach highest throttle value.
End throttle will not be smooth , 
ture_isdbw


3 - if `is_dbw_enabled = True` and didn't wait before switch 
then jerk at start will be very low  , 
Throttle graph shows that we start with good gradual throttle to reach highest value ,
false_isdbw

the best is _situation 1_ ( `is_dbw_enabled = True` )
even-though _situation 3_ will have the best start, but waiting before switching to auto will eliminate all its cons.

**Comment by anasmatic** [comment]
Actually the smooth part at the start is fake _ -when `is_dbw_enable=True`- _ , it is calculated before switching from manual to auto  

still has the same question, any ideas how to smooth throttle start so we won't "spun the wheels"

File: ros/src/twist_controller/dbw_node.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 '''
 
-DEBUG_MODE            = False
+DEBUG_MODE            = True
```
Comment (clockworkbits) [inline ros/src/twist_controller/dbw_node.py:36]: Is this intended?

**Comment by clockworkbits** [comment]
In the walkthrough video there is a lowpass filter with quite low tau applied to the current velocity. Maybe we should try to do similar thing? Not sure if this will help much. But I'm pretty sure we should start with `is_dbw_enabled = False`

**Review by ghost — APPROVED** [review]

**Pull request merged**